Sources/SnapshotTesting/Common/View.swift

```diff
@@ -241,6 +241,74 @@ public struct ViewImageConfig {
     return .init(safeArea: safeArea, size: size, traits: .iPhone13ProMax(orientation))
   }
 
+  public static let iPhone14 = ViewImageConfig.iPhone14(.portrait)
+
+  public static func iPhone14(_ orientation: Orientation) -> ViewImageConfig {
+    let safeArea: UIEdgeInsets
+    let size: CGSize
+    switch orientation {
+    case .landscape:
+      safeArea = .init(top: 0, left: 47, bottom: 21, right: 47)
+      size = .init(width: 844, height: 390)
+    case .portrait:
+      safeArea = .init(top: 47, left: 0, bottom: 34, right: 0)
+      size = .init(width: 390, height: 844)
+    }
+
+    return .init(safeArea: safeArea, size: size, traits: UITraitCollection.iPhone14(orientation))
```

Comment:
I don't think `UITraitCollection.iPhone14(orientation)` compiles because we don't have the iPhone 14 UITraitCollection defined anywhere.